applications/FluidDynamicsApplication/python_scripts/apply_custom_velocity_constraints.py

```diff
@@ -1,5 +1,5 @@
 from KratosMultiphysics import *
-from FluidDynamicsApplication import *
+from KratosMultiphysics.FluidDynamicsApplication import *
```

Comment:
Is this process necessary?
I think the inlet-process can do the same

Reply:
I am not sure. @rubenzorrilla do you know?

Reply:
I think it is legacy code of the very first version of the current GUI. Now we use the processes in the core to assign custom velocity constrains. Feel free to remove it.

Reply:
The file does not seem to be imported anywhere and I could not find any other references to the class inside in the repository, so I went ahead and removed it.